packages/style/src/index.js

```diff
@@ -2,16 +2,46 @@ import React from 'react'
 import niceStyles from 'motion-nice-styles'
 import { StyleSheet, css } from 'aphrodite'
 
-export default function Style(ComposedComponent) {
-  class StyledComponent extends ComposedComponent {
+let opts = {
+  theme: true,
```

Comment:
Default is to have theme but let user write style.configure({ theme: false }) to disable